collections/text-editors/index.md

```diff
@@ -11,6 +11,7 @@ items:
  - leo-editor/leo-editor
  - syl20bnr/spacemacs
  - SpaceVim/SpaceVim
+ - hlissner/doom-emacs
```

Comment:
I think `hlissner/doom-emacs` is an emacs configuration not a distribution like `syl20bnr/spacemacs`.

Reply:
It started out as a personal config yes.

I think the [contributors list](https://github.com/hlissner/doom-emacs/graphs/contributors) shows that it's more than just a configuration.

If you check [init.example.el](https://github.com/hlissner/doom-emacs/blob/develop/init.example.el) we have a modules, instead of layers in Spacemacs.

You can compare installation guides also
https://github.com/syl20bnr/spacemacs#install
Dooms(Since I can't link the header):
Quick start
```git clone https://github.com/hlissner/doom-emacs ~/.emacs.d
~/.emacs.d/bin/doom install```
Doom install just handles the font installation and package downloads instead of on the first load.

We also configure using our own [private modules](https://github.com/Emiller88/doom-emacs-private), rather than in `.emacs.d` like spacemacs `.spacemacs`.